### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by jeqo**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN cd /code && mkdir -p /zipkin-storage-kafka && \
     jar xf *.jar && \
     rm *.jar
 
-FROM openzipkin/zipkin:2.19.1
+FROM openzipkin/zipkin:master
```
Comment (jeqo) [inline docker/Dockerfile]: Seems that `v2.20.2` is not available on docker-hub. It should be updated before merging.

Comment (adriancole) [reply docker/Dockerfile]: @anuraaga hmm this seems to be a pattern

Comment (anuraaga) [reply docker/Dockerfile]: Yeah need to take a deep look at this soon, probably next week.

**Comment by adriancole** [comment]
I will retake this over in a bit

**Comment by jeqo** [comment]
I did test this last week, working fine. But interested on how it looks on your side and if you have some ideas on how the CI pipeline can be improved to give more confidence to merge these changes.

**Comment by jeqo** [comment]
@adriancole sorry for mixing up here. I moved the recommendations from #73 here, as I used this branch as base. 

Regarding java version in travis (15), travis is failing to find it. Will fallback to 14 in the meantime

**Comment by jeqo** [comment]
getting:

```
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception

	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)

	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:324)

	... 49 more

Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container

	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:498)

	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:326)

	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)

	... 50 more

Caused by: com.github.dockerjava.api.exception.NotFoundException: Status 404: {"message":"No such image: openzipkin/zipkin-kafka:latest"}
```
when https://hub.docker.com/layers/openzipkin/zipkin-kafka/latest/images/sha256-d68ddcc5a66504b55596ef5c4c66dd02df32ede14b345b091925eff5ad800e82?context=explore exists 🤔 

are we hitting some limits on docker hub usage?

**Comment by adriancole** [comment]
trying the old "clean cache" trick!

**Comment by adriancole** [comment]
merging in hopes that the travis thing is weirdly related to an old root

**Comment by adriancole** [comment]
@bsideup just in case you've seen this.. it is odd. if I `docker system prune -a` to force re-pull locally, it breaks the same way, but I can pull the image manually and it is fine.. any ideas?

@jeqo we can probably hack by docking docker pull in the travis script temporarily..

**Comment by bsideup** [comment]
@adriancole I will have a look!

**Comment by bsideup** [comment]
It looks like you're hitting https://github.com/testcontainers/testcontainers-java/pull/3313

TL;DR: if you omit `:latest` (explicitly added by Docker CLI), Docker will pull _all_ images for a given image name.
Consider changing this line:
https://github.com/openzipkin-contrib/zipkin-storage-kafka/blob/5bbed7a2ba2cb92f96e27971a6d6c121b405e98a/storage/src/test/java/zipkin2/storage/kafka/ITKafkaStorage.java#L60

and adding `:latest` if you're on rc2 (next version contains a fix but is not released yet)

**Comment by adriancole** [comment]
@bsideup ahah! thanks for the detective work!
